### PR TITLE
Test

### PR DESCRIPTION
**Pull request by emmanuellekamwa**
In this project, we have: 
- Written unit tests for the files `operate.js` and `calculate.js` using Jest.
- Created unit tests for all React components

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mikenath223** [comment]
- [ ] One test case fails kindly check and fix it.

File: src/__tests__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import calculate from '../logic/calculate';
+
+describe('Start tests', () => {
+  test('8 - 4 = 4', () => {
+    let item = {};
+
+    item = { ...item, ...calculate(item, '8') };
```
Comment (mikenath223) [inline src/__tests__/calculate.test.js:7]: - [ ] You have the calculate test in both `tests` and `__tests__` kindly remove one of the test files.

File: src/__tests__/operate.test.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import operate from '../logic/operate';
+
+describe('Supported calculator operators', () => {
+  test('2 + 2 = 4', () => {
+    const a = 2;
+    const b = 2;
```
Comment (mikenath223) [inline src/__tests__/operate.test.js:6]: - [ ] You also have the operate.test.js duplicated in multiple directories; `tests` and `__tests__`, please remove one of the test files.

File: src/__tests__/operate.test.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    expect(answer).toBe('4');
+  });
+
+  test('4 - 1 = 3', () => {
+    const a = 4;
+    const b = 1;
+
+    const answer = operate(a, b, '-');
+
+    expect(answer).toBe('3');
```
Comment (mikenath223) [inline src/__tests__/operate.test.js:19]: - [ ] Kindly include a test case for division by zero.

**Review by mke2111 — APPROVED** [review]
### Hi @emmanuellekamwa & @codecaiine, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
